cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureUtils.java

```diff
@@ -291,6 +292,25 @@ public List<CloudVmInstanceStatus> deallocateInstances(AuthenticatedContext ac,
         return statusesAfterDeallocate;
     }
 
+    @Retryable(backoff = @Backoff(delay = 1000, multiplier = 2, maxDelay = 10000), maxAttempts = 5)
+    public List<CloudVmInstanceStatus> deallocateInstances(AuthenticatedContext ac, List<CloudInstance> vms, Long timeboundInMs) {
+        LOGGER.info("Deallocate VMs: {}", vms.stream().map(CloudInstance::getInstanceId).collect(Collectors.toList()));
```

Comment:
Log line should ideally change to include the timeout.

Reply:
Done